services/brig/schema/src/V70_UserEmailUnvalidated.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2022 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+module V70_UserEmailUnvalidated
+  ( migration,
+  )
+where
+
+import Cassandra.Schema
+import Imports
+import Text.RawString.QQ
+
+migration :: Migration
+migration =
+  Migration 70 "Add email_unvalidated to user table" $
+    schema'
+      [r| ALTER TABLE user ADD (
+          email_unvalidated text
```

Comment:
given that changing this column and changing other columns usually don't coincided (or do they?), wouldn't it be more straight-forward and more readable to have new table?

Reply:
there is one big advantage when having it in a single table, and that is we only have to query C* once when creating/updating the ES index.
if you insist, I'm also ok with having a new table.